projects/plugins/jetpack/_inc/client/security/sso.jsx

```diff
@@ -212,6 +217,11 @@ export const SSO = withModuleSettingsFormHelpers(
 											) }
 										</span>
 									}
+									help={
+										isTwoStepEnforced
+											? __( 'Two-Step Authentication is enforced on this site.', 'jetpack' )
```

Comment:
What would you think about having phrasing that's similar to this one:
https://github.com/Automattic/jetpack/blob/1232e7ece82051eb75d26ebeddd3261b0631d9cf/projects/plugins/jetpack/_inc/client/components/module-toggle/index.jsx#L86-L92

We could also display a link to [the support docs](https://developer.jetpack.com/hooks/jetpack_force_2fa/) in that message.